External/Plugins/ASCompletion/Model/PathModel.cs

```diff
@@ -85,7 +79,6 @@ static public PathModel GetModel(string path, IASContext context)
         public volatile bool Updating;
         public bool WasExplored;
         public bool IsTemporaryPath;
-        public DateTime LastAccess;
```

Comment:
These `LastAccess` and `Touch()` fields are dead code by now, so I went on and removed them.